client/src/game/data/suitsInit.ts

```diff
@@ -92,7 +92,7 @@ export default function suitsInit(COLORS: Map<string, Color>) {
           throw new Error(`The color "${colorString}" in the suit "${suitName}" does not exist.`);
         }
       }
-    } else if (!suitJSON.allClueColors && !suitJSON.noClueColors) {
+    } else if (suitJSON.allClueColors !== true && suitJSON.noClueColors !== true) {
```

Comment:
lol, use === false

Reply:
NOPE: !== true => can be false or undefined. This is exactly the bug!!

Reply:
write a comment then explaining the code that this isn't a bug